temporal-sdk/src/main/java/io/temporal/nexus/OperationHandler.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2022 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ * Copyright (C) 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this material except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.temporal.nexus;
+
+import io.nexusrpc.handler.*;
+import io.temporal.common.Experimental;
+import io.temporal.internal.nexus.CurrentNexusOperationContext;
+import io.temporal.internal.nexus.NexusOperationContextImpl;
+
+/**
+ * Helper to create {@link io.nexusrpc.handler.OperationHandler} instances that take a {@link
+ * io.temporal.client.WorkflowClient}.
+ */
+@Experimental
+public final class OperationHandler {
```

Comment:
Will there ever be a day this may be an interface? Trying to get an idea of the spread of static methods here.

Reply:
No, this class is just to hold functions because Java doesn't have top level functions

Reply:
I wonder if `Nexus.syncClientHandler` would make more sense

Reply:
The `Nexus` class if for functions that can be used **inside** an operation , we have an equivalent class for workflows and activities

Reply:
But this is a class called operation handler that is not an operation handler and causes ambiguity with the actual nexus operation handler. There are a few of these static-method-older things (ref https://github.com/temporalio/sdk-java/pull/2231#discussion_r1771313880). If at all possible we should consider consolidating some, even if not using the `Nexus` class.